package.json

```diff
@@ -25,10 +25,12 @@
     "start": "grunt dev",
     "test": "grunt test",
     "docs": "npm run docs:lint && npm run docs:api",
-    "docs:api": "jsdoc -r src/js -d docs/api -c .jsdoc.json",
+    "jsdoc": "jsdoc",
```

Comment:
This is like the `grunt` task above, it's so that you can more easily run `jsdoc` commands on the cli without installing it globally if `jsdoc -c .jsdoc.json` doesn't suit what you're trying to do.